composer.json

```diff
@@ -49,7 +49,8 @@
   "config": {
     "allow-plugins": {
       "simplesamlphp/composer-module-installer": true,
-      "yiisoft/yii2-composer": true
+      "yiisoft/yii2-composer": true,
+      "simplesamlphp/composer-xmlprovider-installer": true
```

Comment:
Why was this added? I'm guessing it was automatically added, but it can probably be removed or changed to `false` since we don't use it as far as I can tell.

Reply:
It was automatically updated.